<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatorWindow</class>
 <widget class="QWidget" name="CreatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>726</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QGroupBox" name="groupBoxMachineInfo">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>999999</height>
          </size>
         </property>
         <property name="title">
          <string>机床信息</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLineEdit" name="lineEditMachineName">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>demo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditMachineNameLng">
              <property name="text">
               <string>demo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditMachineIndex">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditCustomInfo">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxLng">
              <item>
               <property name="text">
                <string>中文</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>多语言</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="labelMachineName">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>机床型号</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelMachineNameLng">
              <property name="text">
               <string>机床名称</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelMachineIndex">
              <property name="text">
               <string>机床编号</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelCustomInfo">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>用户名称</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelLng">
              <property name="text">
               <string>机床语言</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>4444444</height>
          </size>
         </property>
         <property name="title">
          <string>其他设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="labelIfRemoveComments">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>删除注释</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxIfRemoveComments">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>是</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>否</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelHMIMode">
            <property name="text">
             <string>界面模式</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBoxHMIMode">
            <item>
             <property name="text">
              <string>标准</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>兼容</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelSystemMode">
            <property name="text">
             <string>数控系统</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBoxSystemMode">
            <item>
             <property name="text">
              <string>非 ONE 系统</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sinumerik ONE</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelLoadingArm">
            <property name="text">
             <string>机械手</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="comboBoxLoadingArm">
            <item>
             <property name="text">
              <string>无</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>有</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>机床配置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="labelMachineType">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>机床类型</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelIfHasScrewTap">
            <property name="text">
             <string>是否磨丝锥</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelWareType">
            <property name="text">
             <string>修整器类型</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelIfHasWorm">
            <property name="text">
             <string>是否磨蜗杆</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelWheelType">
            <property name="text">
             <string>修整轮类型</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelShapeType">
            <property name="text">
             <string>齿形选择</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="labelIfHasReOp">
            <property name="text">
             <string>二次对刀</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="labelIfOperation">
            <property name="text">
             <string>自动对刀</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="labelMeasureMethord">
            <property name="text">
             <string>对刀机构</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="labelProbePos">
            <property name="text">
             <string>测量方向</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="labelIfCenter">
            <property name="text">
             <string>磨削中心</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="labelIfHasU">
            <property name="text">
             <string>U轴</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="labelIfNotHasUExternalPos">
            <property name="text">
             <string>外磨位置</string>
            </property>
           </widget>
          </item>
          <item row="13" column="0">
           <widget class="QLabel" name="labelIfHasA">
            <property name="text">
             <string>A轴</string>
            </property>
           </widget>
          </item>
          <item row="14" column="0">
           <widget class="QLabel" name="labelGrindWheelType">
            <property name="text">
             <string>砂轮驱动</string>
            </property>
           </widget>
          </item>
          <item row="15" column="0">
           <widget class="QLabel" name="labelExGrindWheelType">
            <property name="text">
             <string>外磨砂轮驱动</string>
            </property>
           </widget>
          </item>
          <item row="16" column="0">
           <widget class="QLabel" name="labelDressWheelType">
            <property name="text">
             <string>修整轮驱动</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QComboBox" name="comboBoxMachineType">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>外螺纹</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>内螺纹</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="comboBoxIfHasScrewTap">
            <item>
             <property name="text">
              <string>是</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>否</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QComboBox" name="comboBoxWareType">
            <item>
             <property name="text">
              <string>VW</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>V</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>XZ</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>X_Back</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>液压</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>XW</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QComboBox" name="comboBoxIfHasWorm">
            <item>
             <property name="text">
              <string>否</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>是</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QComboBox" name="comboBoxWheelType">
            <item>
             <property name="text">
              <string>单/方滚轮</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>双滚轮</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QComboBox" name="comboBoxShapeType">
            <item>
             <property name="text">
              <string>全套齿形</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>双圆弧</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QComboBox" name="comboBoxIfHasReOp">
            <item>
             <property name="text">
              <string>有</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>无</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QComboBox" name="comboBoxIfOperation">
            <item>
             <property name="text">
              <string>有</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>无</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QComboBox" name="comboBoxMeasureMethord">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>开关</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>测头</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>开关和测头</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QComboBox" name="comboBoxProbePos">
            <item>
             <property name="text">
              <string>向前测量</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>向后测量</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QComboBox" name="comboBoxIfCenter">
            <item>
             <property name="text">
              <string>是</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>不是</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QComboBox" name="comboBoxIfHasU">
            <item>
             <property name="text">
              <string>有</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>无</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QComboBox" name="comboBoxIfNotHasUExternalPos">
            <item>
             <property name="text">
              <string>后置</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>前置</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="13" column="0">
           <widget class="QComboBox" name="comboBoxIfHasA">
            <item>
             <property name="text">
              <string>有</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>无</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="14" column="0">
           <widget class="QComboBox" name="comboBoxGrindWheelType">
            <item>
             <property name="text">
              <string>伺服</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>变频器</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="15" column="0">
           <widget class="QComboBox" name="comboBoxExGrindWheelType">
            <item>
             <property name="text">
              <string>伺服</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>变频器</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="16" column="0">
           <widget class="QComboBox" name="comboBoxDressWheelType">
            <item>
             <property name="text">
              <string>伺服</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>变频器</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="labelInfo">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelProcessing">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
     <property name="spacing">
      <number>100</number>
     </property>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="saveDataPushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>保存设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="creatPushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>程序生成</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
